src/js/utils/SystemLogUtil.js

```diff
@@ -0,0 +1,44 @@
+const SystemLogUtil = {
+  getUrl(nodeID, options, isStream = true) {
+    let {params = {}} = options;
+
+    let range = ['cursor', 'limit', 'skip_prev', 'skip_next'].reduce(function (memo, key) {
```

Comment:
Changed them into more simple reducers, don't see a need for them to be a general url util, other than the reducer part